lib/core/cache/tweet_cache.dart

```diff
@@ -0,0 +1,62 @@
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:harpy/api/twitter/data/tweet.dart';
+import 'package:harpy/core/app_configuration.dart';
+import 'package:harpy/core/filesystem/cache_dir_service.dart';
+import 'package:logging/logging.dart';
+
+class TweetCache {
+  final Logger log = new Logger('TweetCache');
+
+  CacheDirectoryService _cacheDirService;
```

Comment:
```suggestion
  CacheDirectoryService _cacheDirService = CacheDirectoryService();
```
anstatt dass im constructor zu initialisieren

Reply:
see above :)